src/configs/index.js

```diff
@@ -23,114 +23,9 @@ export const ABOUT_US_IMAGES = {
   workPlace_2: './images/about-us/work-place-2.jpg'
 };
 
-export const CABINET_OPTIONS_NOT_LOGGED = {
```

Comment:
Please, create object called `LANGUAGE_MAP` where the name is a locale of language and the value is number. And whenever you use objects for translations instead of plain numbers use `LANGUAGE_MAP.en` or `LANGUAGE_MAP.ua`